src/platform/silabs/rs911x/rsi_ble_config.h

```diff
@@ -72,21 +83,35 @@
 #define RSI_BLE_NAME_SPACE (0x01)
 #define RSI_BLE_DESCRIPTION (0x010B)
 
-#define RSI_BLE_MAX_NBR_ATT_REC (80)
-
 //! BLE characteristic custom service uuid
 #define RSI_BLE_CUSTOM_SERVICE_UUID (0xFFF6)
 #define RSI_BLE_CUSTOM_LEVEL_UUID (0x1FF1)
-#define RSI_BLE_TX_OCTETS 251
-#define RSI_BLE_TX_TIME 2120
-#define RSI_BLE_MATTER_CUSTOM_SERVICE_DATA_LENGTH 240
+
+#ifdef RSI_M4_INTERFACE
+#define RSI_BLE_MAX_NBR_ATT_REC (20)
+
+#if (SIWX_917 | EXP_BOARD)
+#define RSI_BLE_MAX_NBR_PERIPHERALS (1)
+#else
+#define RSI_BLE_MAX_NBR_SLAVES (1)
+#endif
+
+#define RSI_BLE_NUM_CONN_EVENTS (2)
+#else
+#define RSI_BLE_MAX_NBR_ATT_REC (80)
 
 #if (SIWX_917 | EXP_BOARD)
-#define RSI_EXT_TCPIP_FEATURE_BITMAP 0
-#define CLEAR_ACCEPTLIST (0x00)
-#define ADD_DEVICE_TO_ACCEPTLIST (0x01)
-#define DELETE_DEVICE_FROM_ACCEPTLIST (0x02)
 #define RSI_BLE_MAX_NBR_PERIPHERALS (3)
+#else
+#define RSI_BLE_MAX_NBR_SLAVES (3)
+#endif
+
+#define RSI_BLE_NUM_CONN_EVENTS (20)
+#endif

```

Comment:
This series of conditional definitions is a bit convoluted. 

```suggestion
#endif // RSI_M4_INTERFACE
```


We will have to think of something clearer when you get the chance. maybe separate them and each wifi board has a ble_config in matter support or something like that